src/Microsoft.Azure.VirtualKubelet.Edge.Provider/EdgeProvider.cs

```diff
@@ -201,7 +201,7 @@ private static void AddModuleTwin<T>(Configuration configuration, string key, T
                             RestartPolicy = "always",
                             Settings = new EdgeModuleSettings
                             {
-                                Image = "microsoft/azureiotedge-agent:1.0-preview",
+                                Image = "mcr.microsoft.com/azureiotedge-agent:1.0",
```

Comment:
I am wondering if it'll be better to do something along the lines of:
```csharp
Image = Environment.GetEnvironmentVariable("EDGE_AGENT_IMAGE") ?? "mcr.microsoft.com/azureiotedge-agent:1.0"
```
For both Edge Agent and Edge Hub. This way this can be overridden in a specific deployment if need be (without us needing to build new Docker images for the provider).

Reply:
My PR was just a short time fix to win some time implementing a more valuable solution. But this sounds better to be flexible without building new Docker images for the provider.